v19.2/unsplit-at.md

```diff
@@ -0,0 +1,192 @@
+~~~
+~~~
+  range_id |          start_key           |      start_pretty      |           end_key            |       end_pretty       | database_name | table_name | index_name | replicas | learner_replicas | split_enforced_until | lease_holder
++----------+------------------------------+------------------------+------------------------------+------------------------+---------------+------------+------------+----------+------------------+----------------------+--------------+
+        21 | \275                         | /Table/53              | \275\211\022chicago\000\001  | /Table/53/1/"chicago"  | movr          | users      |            | {1,2,3}  | {}               | NULL                 |            1
+        27 | \275\211\022chicago\000\001  | /Table/53/1/"chicago"  | \275\211\022new york\000\001 | /Table/53/1/"new york" | movr          | users      |            | {1,2,3}  | {}               | NULL                 |            3
+        28 | \275\211\022new york\000\001 | /Table/53/1/"new york" | \275\211\022seattle\000\001  | /Table/53/1/"seattle"  | movr          | users      |            | {1,2,3}  | {}               | NULL                 |            2
+        29 | \275\211\022seattle\000\001  | /Table/53/1/"seattle"  | \276                         | /Table/54              | movr          | users      |            | {1,2,3}  | {}               | NULL                 |            1
+(4 rows)
+~~~
+
+The `split_enforced_until` column is now `NULL` for all ranges in the `users` table. The table is still split into ranges at `'chicago'`, `'new york'`, and `'seattle'`, but the split is no longer enforced.
+
+### Unsplit an index
+
+Add a new secondary [index](indexes.html) to the `rides` table, on the `revenue` column, and then split the table ranges by secondary index values:
+
+{% include copy-clipboard.html %}
+~~~ sql
+> CREATE INDEX revenue_idx ON rides(revenue);
+~~~
+
+{% include copy-clipboard.html %}
+~~~ sql
+> ALTER INDEX rides@revenue_idx SPLIT AT VALUES (25.00), (50.00), (75.00);
+~~~
+~~~
+         key        |      pretty      |       split_enforced_until
++-------------------+------------------+----------------------------------+
+  \277\214*2\000    | /Table/55/4/25   | 2262-04-11 23:47:16.854776+00:00
+  \277\214*d\000    | /Table/55/4/5E+1 | 2262-04-11 23:47:16.854776+00:00
+  \277\214*\226\000 | /Table/55/4/75   | 2262-04-11 23:47:16.854776+00:00
+(3 rows)
+~~~
+
+{% include copy-clipboard.html %}
+~~~ sql
+> SELECT * FROM crdb_internal.ranges WHERE table_name='rides';
+~~~
+~~~
+  range_id |     start_key     |   start_pretty   |      end_key      |    end_pretty    | database_name | table_name | index_name  | replicas | learner_replicas |       split_enforced_until       | lease_holder
++----------+-------------------+------------------+-------------------+------------------+---------------+------------+-------------+----------+------------------+----------------------------------+--------------+
```

Comment:
Note that we have more updates to do re: `split_enforced_until` incoming via https://github.com/cockroachdb/cockroach/pull/38004 (namely a WITH EXPIRATION option to SPLIT AT) but that will happen via another ticket.